panoptes_aggregation/reducers/tess_gold_standard_reducer.py

```diff
@@ -25,7 +25,8 @@ def process_data(extracts):
     success = []
     for extract in extracts:
         if extract['feedback']:
-            success.append([transit['success'] for transit in extract['feedback']])
+            li = [transit['success'] for transit in extract['feedback']]
+            success.append(li)
```

Comment:
Was this change for readability?  Happy for it to stay, just wondering what prompted it.